test/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This is not a folder for unit tests, but a folder for things needed for testing such as the bootstrap.js file. I'm not sure this is the best name for the folder, but I could not think of a better one.

Reply:
Yes, this is potentially a little confusing. We don't have an alternate convention to apply, so I think we can stick with this for the moment.